build_mt.py

```diff
@@ -404,61 +391,33 @@ def main(sim_name, z_start, z_stop, merger_parent, catalog_parent, resume=False,
         # reorder file names by super slab number
         fns_this = reorder_by_slab(fns_this,minified)
         fns_prev = reorder_by_slab(fns_prev,minified)
+        
+        # maybe we want to support resuming from arbitrary superslab
+        first_ss = 0
+        
+        # We're going to be loading slabs in a rolling fashion:
+        # reading the "high" slab at the leading edge, discarding the trailing "low" slab
+        # and moving the mid to low. But first we need to read all three to prime the queue
+        mt_prev = {}  # indexed by slab num
+        mt_prev[(first_ss-1)%n_chunks] = get_mt_info(fns_prev[(first_ss-1)%n_chunks], fields=fields_mt, minified=minified)
+        mt_prev[first_ss] = get_mt_info(fns_prev[first_ss], fields=fields_mt, minified=minified)
 
         # for each chunk
-        for k in range(n_chunks):
-
-            # starting and finishing superslab chunks; 
-            inds_fn_this = [k]
-            inds_fn_prev = np.array([(k-1),k,(k+1)],dtype=int) % n_chunks
+        for k in range(first_ss,n_chunks):
+            # starting and finishing superslab chunks
+            klow = (k-1)%n_chunks
+            khigh = (k+1)%n_chunks
```

Comment:
`klow` and `khigh` are the leading and trailing chunks in the previous redshift.  Of course, the current redshift is only looking at one chunk, `k`.